functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +268,37 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
-										
+
+					if($lastbackup[0].Path.StartsWith('\\') -eq $false -and $CopyDestination -eq $true) 
```

Comment:
This is valid, but you can do `$CopyDestination` in place of `$CopyDestination -eq $true` - we're going in that direction with 1.0